server/command_hooks.go

```diff
@@ -522,6 +534,50 @@ func (p *Plugin) crash() {
 	_ = 1 / y
 }
 
+func (p *Plugin) executeCommandMentions(args *model.CommandArgs) *model.CommandResponse {
+	message := "The command `" + args.Command + "` contains the following different mentions.\n"
+	message += "### Mentions to users in the team\n"
+	if args.UserMentions == nil {
+		message += "_There are no mentions to users in the team in your command_.\n"
+	} else {
+		message += "| User name | Unique identifier |\n"
```

Comment:
Nit: `ID` is the commonly used at Mattermost 
```suggestion
		message += "| User name | ID |\n"
```